compiler/src/dotty/tools/dotc/parsing/Tokens.scala

```diff
@@ -285,7 +285,7 @@ object Tokens extends TokensCommon {
 
   final val endMarkerTokens = identifierTokens | BitSet(IF, WHILE, FOR, MATCH, TRY, NEW, THROW, GIVEN, VAL, THIS)
 
-  final val skipStopTokens = BitSet(SEMI, NEWLINE, NEWLINES, RBRACE, RPAREN, RBRACKET, OUTDENT)
+  final val skipStopTokens = BitSet(SEMI, NEWLINE, NEWLINES, RBRACE, RPAREN, RBRACKET, OUTDENT, COMMA)
```

Comment:
Possible controversial. This makes it so that `foo(bar, <some error>, baz)` still properly parses `baz`.

Reply:
I'm not sure what the downsize of add it is though. Only two tests were affected, and IMHO the extra errors are an improvement.